src/app.ts

```diff
@@ -133,12 +140,14 @@ function addCustomCSS(windowElement: BrowserWindow): void {
   }
 }
 
-app.on('ready', () => {
-  createWindow()
+;(async () => {
```

Comment:
Why this change?

Reply:
Actually not really needed. I just saw in the `electron-dl` example that `app.whenReady()` can be used as a convenient alternative. I can revert this.

Reply:
Yeah, I think the `app.on('ready', () => ...)` is cleaner and fits better with the rest of the app.

Reply:
Reverted, thanks for bringing this up.